aws_jobs/cyhy-data-extract.py

```diff
@@ -380,6 +380,16 @@ def main():
             },
             "projection": default_projection,
         },
+        "snapshots": {
```

Comment:
Move the `snapshots` section below the `requests` section to keep things alphabetized.

Reply:
Done in commit 816a23a.